### ISSUE TITLE
Don't render a Table of Contents for a one-item book

### ISSUE THREAD

**Issue by slifty** [OpenTechStrategies/SimpleBook#53]
If a book only has one item I would like to suggest that the Table of Contents should not be rendered.

Who's with me?

**Comment by slifty** [comment]
One related use case: When #52 is fixed, we will have one-item books regularly being generated.  Those books will generally also NOT have titles or subtitles (right now if there is no title provided, the title page is not rendered).  In that use case we would currently generate a table of contents with 1 item, and then the item.

**Comment by jim-mcgowan** [comment]
@slifty: I don't have a problem with not rendering a TOC for one item books, however, it seems like it would be more work to exclude it.

**Comment by kfogel** [comment]
I agree, it would look weird to have a TOC for a one-item book, and I think it's worth the (surely very small?) amount of extra work to exclude it.

**Comment by slifty** [comment]
Excellent!

This is a very small change indeed.
